README.md

```diff
@@ -212,6 +212,14 @@ By default setup-minikube caches the ISO, kicbase, and preload using GitHub Acti
   </pre>
 </details>
 
+<details>
+  <summary>bin-path (optional)</summary>
```

Comment:
thank you for this PR, how about a more descriptive name, such as "install-path" ?

Reply:
Hey, sorry - I oversaw the notification. Thanks for the suggestion, sounds better (I think I just took over the existing binPath variable name and exposed it...).  [ad9137a](https://github.com/medyagh/setup-minikube/pull/214/commits/ad9137aefa65986c0f2a954394ba4ac61afd8f7d) renames the arg to `install-path`.